lib/ai/providers/gemini-provider.ts

```
@@ -1,41 +1,54 @@
+  }
+
+  async generateEmbedding(value: string): Promise<number[]> {
+    const model = this.client.getGenerativeModel({
+      model: AI_MODELS.GEMINI.EMBEDDING,
+    });
+
+    try {
+      const result = await model.embedContent(value);
+      return result.embedding.values;
+    } catch (err) {
+      throw err;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)